src/integrationTest/stepFunctions/visualizeStateMachine.test.ts

```diff
@@ -95,6 +96,27 @@ async function waitUntilWebviewIsVisible(webviewPanel: vscode.WebviewPanel | und
     })
 }
 
+/**
+ * Executes the close all editors command but also waits for the active editor to disappear
+ *
+ */
+async function closeAllEditors() {
```

Comment:
nice change! Let's move it to a util module, e.g. `src/shared/utilities/vsCodeUtils.ts` . Then it can be used by sam.test.ts too, which has its own `closeAllEditors` function that can now be removed :)